client/verta/verta/_deployment/endpoint.py

```diff
@@ -126,13 +126,22 @@ def _get_json_by_path(cls, conn, workspace, path):
                 return endpoint
         return None
 
+    env_vars_err_msg = "env_vars must be (string, string) dictionary"
+
     def update(self, run, strategy, wait=False, resources=None, autoscaling=None, env_vars=None):
         if not isinstance(run, experimentrun.ExperimentRun):
             raise TypeError("run must be an ExperimentRun")
 
         if not isinstance(strategy, _UpdateStrategy):
             raise TypeError("strategy must be an object from verta.deployment.strategies")
 
+        if env_vars:
+            if not isinstance(env_vars, dict):
+                raise TypeError(self.env_vars_err_msg)
+            for key, value in env_vars.items():
+                if not isinstance(key, str) or not isinstance(value, str):
```

Comment:
Because Python 2 and Python 3 have multiple string types 😞 
Will also require `from ..external import six` near line 8.
```suggestion
                if not isinstance(key, six.string_types) or not isinstance(value, six.string_types):
```

Reply:
added import and your change